### ISSUE TITLE
Using type safe plugins instead of hardcoded strings in build.gradle(s)

### ISSUE THREAD

**Issue by VahidGarousi** [Kaaveh/ComposeNews#238]
![image](https://github.com/user-attachments/assets/a8cc34c4-c972-47ea-87cc-0de6ee55dcf2)

**Comment by AzMostafa** [comment]
Hey @VahidGarousi,

Just to clarify — do you mean using a Version Catalog (i.e. libs.versions.toml) for this issue?

**Comment by VahidGarousi** [comment]
> Hey @VahidGarousi,
> 
> Just to clarify — do you mean using a Version Catalog (i.e. libs.versions.toml) for this issue?

Yes, exactly — and also make sure to Ctrl + F and replace all instances to use type-safe access through the version catalog (i.e. libs.<alias>).

**Comment by AzMostafa** [comment]
Got it, thanks!

I'll start working on migrating all dependencies and plugins to use the type-safe accessors via the version catalog (libs.).
I'll make sure to search and replace all the hardcoded references as well.

**Comment by AzMostafa** [comment]
Hey again @VahidGarousi,

can you assigned me for this issue?

**Comment by VahidGarousi** [comment]
> Hey again [@VahidGarousi](https://github.com/VahidGarousi),
> 
> can you assigned me for this issue?

Of course! Just assigned you. Thanks for taking it on 🙌
